src/MOI_wrapper.jl

```diff
@@ -12,7 +12,7 @@ struct Optimizer{T,M} <: MOI.AbstractOptimizer
         )
     end
     function Optimizer{T}(optimizer_constructor) where {T}
-        return Optimizer(Context{T}(optimizer_constructor))
+        return Optimizer(Context{T}(optimizer_constructor; add_cache = true))
```

Comment:
Can you add a reference to the MOI issue ? I'm not sure I will remember if I read it a year from now